playbooks/ha_log_cfg_sync.yaml

```diff
@@ -0,0 +1,138 @@
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+---
+- name: Config rsync daemon on log server slave node
+  hosts: logserver-slave, zuul-scheduler-slave
```

Comment:
these tow nodes are the same one, will the task be executed twice?

Reply:
it is only in *this* deployment scenario, that the log server and zuul scheduler are on same host. we'd better do not limit to this situation, to make this tool more general. in case that we will deploy log server in separated host.

Reply:
if so, L77-L96 can be merged into this section as well.

And if log server and zuul server are different hosts, the job at L35 will be run on both hosts which will create both zuul-config and log sync job, it'll be wrong.

right?

Reply:
yes, you are right, will change, thx

Reply:
> Just a question: will the task be executed twice in same host?

no, ansible will handle all the groups to hosts before running the tasks,  you can specify more different groups in a play, but if they only contain 1 host, it will only run once on the host.

Reply:
> > Just a question: will the task be executed twice in same host?
> 
> no, ansible will handle all the groups to hosts before running the tasks, you can specify more different groups in a play, but if they only contain 1 host, it will only run once on the host.

Thank your clarification.